Content.Server/GameObjects/Components/Fluids/PuddleComponent.cs

```diff
@@ -49,7 +50,7 @@ public class PuddleComponent : Component
         public override string Name => "Puddle";
 
         private CancellationTokenSource _evaporationToken;
-        private ReagentUnit _evaporateThreshold; // How few <Solution Quantity> we can hold prior to self-destructing
+        public ReagentUnit _evaporateThreshold; // How few <Solution Quantity> we can hold prior to self-destructing
```

Comment:
If you're gonna make this public, encapsulate it in a property or auto-property. Also this isn't following naming standards. It should be "EvaporateThreshold". I'd personally keep this private and make a property.